### PR TITLE
gl_rasterizer: Use GPU virtual addr when possible for framebuffers.

### PR DESCRIPTION
**Pull request by bunnei**

### CODE REVIEW

File: src/core/hle/service/nvdrv/devices/nvhost_as_gpu.cpp [hunk 1]
```diff
@@ -85,8 +85,8 @@ u32 nvhost_as_gpu::Remap(const std::vector<u8>& input, std::vector<u8>& output)
         u64 size = static_cast<u64>(entry.pages) << 0x10;
         ASSERT(size <= object->size);
 
-        Tegra::GPUVAddr returned = gpu.memory_manager->MapBufferEx(object->addr, offset, size);
-        ASSERT(returned == offset);
+        object->gpu_addr = gpu.memory_manager->MapBufferEx(object->cpu_addr, offset, size);
```
Comment (Subv) [inline src/core/hle/service/nvdrv/devices/nvhost_as_gpu.cpp]: You set the GPU address here and in MapBufferEx, but you don't unset it when Unmap is called

Comment (Subv) [reply src/core/hle/service/nvdrv/devices/nvhost_as_gpu.cpp]: Also, the logic is wrong here, MapBufferEx and Remap can be called multiple times with the same nvmap object at different offsets and it'll return different addresses, right now you're just overwriting the gpu address to whatever was used last

**Comment by Hexagon12** [comment]
Please fix conflicts kthx

**Comment by Subv** [comment]
I think this PR should remove the first and last commits, leaving only the ` gl_rasterizer_cache: Track page counts in CPU virtual addresses. ` commit.

**Comment by Hexagon12** [comment]
Is this PR still relevant?

**Pull request closed without merging**